### DISCUSSION TITLE
Deploying pages and or endpoints as Cloud Functions

### DISCUSSION THREAD

**Discussion by kireerik** [withastro/roadmap#584] [Proposal]
### Body

@sarah11918 Can we deploy (Astro) pages and or endpoints as [Cloud Functions for Firebase](https://firebase.google.com/products/functions) (same as [Google Cloud Functions](https://cloud.google.com/functions))?

**Comment by kireerik** [comment]
@sarah11918 What do you think?

**Comment by kireerik** [comment]
@jasikpark What do you think about this?

**Comment by kireerik** [comment]
@Princesseuh What do you think about this?

**Comment by Princesseuh** [comment]
Please avoid spamming. If people knew, or had an opinion, they would've answered already.
